src/__tests__/mockApiResponse.json

```diff
@@ -0,0 +1,103 @@
+{
+  "collection": {
```

Comment:
A good chunk of the data in this file could be stripped out, Id be tempted to remove anything not being used elsewhere to cut down the noise a bit